### ISSUE TITLE
progress stop at 99.99% of this epoch

### ISSUE THREAD

**Issue by qnkhuat** [Bartzi/see#15]
<img width="1552" alt="screen shot 2018-03-18 at 6 23 40 pm" src="https://user-images.githubusercontent.com/25661381/37565302-80baaea0-2ad9-11e8-978b-c474b194a66e.png">
its stayed like this for 10m.
Is there any problem with it ?

**Comment by Bartzi** [comment]
No, there is no problem. The program is performing a validation of your trained model on the entire validation dataset. This will take a while. Everything is alright.

**Comment by Bartzi** [comment]
I'm also seeing an `nan` in your train log. Did you adjust the learning rate to a lower value?

**Comment by qnkhuat** [comment]
I didn't change anything.  btw I encourage the same problem in another remote. the whole losses is nan.
<img width="528" alt="screen shot 2018-03-19 at 9 14 13 pm" src="https://user-images.githubusercontent.com/25661381/37600651-81771bc6-2bba-11e8-9eb5-75c4610393f1.png">

**Comment by Bartzi** [comment]
In this case you definitely need to adjust to learning rate to `1e-4` or `1e-5`.

**Comment by qnkhuat** [comment]
it didn't help. I still receive nan

**Comment by Bartzi** [comment]
it could be that a division by zero occurs somewhere... If adjusting the learning rate does not help, you could check for that and use chainer in [debug mode](https://docs.chainer.org/en/stable/reference/core/debug.html?highlight=debug).

**Comment by qnkhuat** [comment]
Its yielded : Exception in main training loop: Each label `t` need to satisfy `0 <= t < x.shape[1] or t == -1`;
Concretely:
<img width="718" alt="screen shot 2018-03-21 at 12 42 19 am" src="https://user-images.githubusercontent.com/25661381/37672552-d01c8900-2ca0-11e8-8dc7-800d9a41d9bc.png">

It is funny that I used debug mode on another machine (which don't have nan loss) it also yields the same.

**Comment by Bartzi** [comment]
Seems the shapes produced by the network are not as they should be. Are you using your own data?

**Comment by qnkhuat** [comment]
Yes. I've created my own data. 
I trained it on another machine and it doesn't get the nan. 
But it stuck at 99.96% for a day :D

**Comment by Bartzi** [comment]
Then you should check the number of classes your dataset has. Did you adjust the network, to fit to your number of classes?

How large is your validation set?

**Comment by qnkhuat** [comment]
I need to detect 1 text with 17 chars.

`17      1
$PATH   1GCHTCFE4C8101563`
Example of my gt

My validation set is 1.2Gb. Is it too big?

**Comment by Bartzi** [comment]
How many different characters do you want to recognize?

3700 images is not to much for validation. Actually it should work... I'm not sure why it doesn't. You can, however, just uncomment the epoch evaluator from the training script and then this should not be a problem anymore.

**Comment by qnkhuat** [comment]
yea. But it still receives nan :(.

**Comment by vbansal86** [comment]
Hi did u resolve this issue? i am facing same
